sql/localsql/events/events.go

```diff
@@ -5,17 +5,24 @@ import (
 	"fmt"
 	"time"
 
+	spacemeshv2alpha1 "github.com/spacemeshos/api/release/go/spacemesh/v2alpha1"
+
 	"github.com/spacemeshos/go-spacemesh/common/types"
 	"github.com/spacemeshos/go-spacemesh/sql"
+	"github.com/spacemeshos/go-spacemesh/sql/builder"
 )
 
-func InsertEvent(db sql.Executor, id types.NodeID, timestamp time.Time, eventBytes []byte) error {
+func InsertEvent(
+	db sql.Executor, id types.NodeID, timestamp time.Time, kind spacemeshv2alpha1.IdentityState, eventBytes []byte,
+) error {
```

Comment:
I'm not sure it's a good idea to import the proto definition for the `kind` here. How about using just an integer?